tests/fuzz/csmith_driver.py

```diff
@@ -143,6 +143,7 @@ def fail():
     notes['embug'] += 1
     fails += 1
     shutil.copyfile(fullname, 'newfail_%d_%d%s' % (os.getpid(), fails, suffix))
+    sys.exit(1)
```

Comment:
it's intentional that it continues after failures. this allows running it overnight and gathering a bunch of errors.